sci-libs/libqalculate/libqalculate-4.6.1.ebuild

```diff
@@ -0,0 +1,79 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+# Bump with sci-calculators/qalculate-gtk!
```

Comment:
The comment suggests we should bump `qalculate-gtk` at the same time, could you add a matching version for that package as well?